Sources/WebDriver/Session.swift

```diff
@@ -73,6 +73,14 @@ public class Session {
             Requests.SessionTimeouts(session: id, type: type, ms: duration * 1000))
     }
 
+    public func synchronous(javascript script: String,  args: [String]) throws {
+        try webDriver.send(Requests.SessionScript(session: id, script: script, args: args))
+    }
+
+    public func asynchronous(javascript script: String,  args: [String]) throws {
```

Comment:
I think it would be more ergonomic to have a single function with an `async: Bool` parameter